backend/django_core/templates/anime/_layout.html

```diff
@@ -142,11 +142,11 @@
                 </aside>
                 <div id="page" class="flex-1 overflow-x-hidden flex flex-col" style="scrollbar-gutter: auto;">
                     {% if request.path == "/anime/" %}
-                        {% include 'anime/_partial_home.html'%}
+                        {% include 'anime/_partials/home.html'%}
                     {% elif request.path == "/anime/explore/" %}
-                        {% include 'anime/_partial_explore.html'%}
+                        {% include 'anime/_partials/explore.html'%}
                     {% elif request.resolver_match.view_name == 'anime_info_view' %}
-                        {% include 'anime/_partial_info.html'%}
+                        {% include 'anime/_partials/info.html'%}
```

Comment:
I dont think this is the better way of doing things. Check the register/login page for how i implemented things

Reply:
> Check the register/login page for how i implemented things

yea I tried that but reverted, uhh I forgot the reason
you mean **extending** right?

Reply:
NOOO . Check the register page lgoic

Reply:
Yeaa got it :sparkles:

Reply:
https://github.com/baseplate-admin/CoreProject/blob/93662b1b3b163591964af248b653efff2e16756b/backend/django_core/templates/user/register/index.html#L18-L23